### DISCUSSION TITLE
3DEP resolutions and update frequency

### DISCUSSION THREAD

**Discussion by scottyhq** [microsoft/PlanetaryComputer#214] [Q&A]
That dataset page for [3DEP Seamless DEMs on Planetary Computer](https://planetarycomputer.microsoft.com/dataset/3dep-seamless#overview) states DEMs are available "at approximately 1, 3, and 9 meters for parts of the United States." But it looks like PC only hosts 10 and 30m GSD and the temporal extent is 01/01/1925 – 05/06/2020. Are there plans to add 1m GSD data?

For datasets like this that are "updated continuously to integrate newly available, improved elevation source data" (https://www.sciencebase.gov/catalog/item/4f70a58ce4b058caae3f8ddb) it would be convenient to have a "Last Updated" field on the Data Catalog page.

**Comment by TomAugspurger** [comment]
The pipeline for that data is a bit stale. We don't *currently* have plans to update that, but I'll post here when we do.

> it would be convenient to have a "Last Updated" field on the Data Catalog page.

I see how that would be helpful. I'll need to think on it a bit implementation-wise (by design, at the database level our collections don't really know anything about their items). In the meantime, the `/<collection>/items?limit=1` is how I typically check the newest item in a colleciton.
